### PR TITLE
x11-misc/gmrun: Add 0.9.2-r2 w/ gcc-6 pie patch

### PR DESCRIPTION
**Pull request by lucianposton**
r2 is updated to eapi 6 and includes the patch to fix https://bugs.gentoo.org/640018

### CODE REVIEW

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils
```
Comment (Zlogene) [inline x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: I do not see how `eutils` eclass serves you here :)

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+"
+DEPEND="
+	${RDEPEND}
+	elibc_glibc? ( >=sys-libs/glibc-2.10 )
+	sys-apps/sed
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	epatch \
```
Comment (Zlogene) [inline x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: it is `eapply` with EAPI=6, not `epatch`

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		"${FILESDIR}"/${P}-gcc6-pie.patch \
+		"${FILESDIR}"/${P}-sysconfdir.patch \
+		"${FILESDIR}"/${P}-glibc210.patch \
+		"${FILESDIR}"/${P}-stlport.patch
+
+	eapply_user
+	eautoreconf
+}
+
+src_install() {
```
Comment (Zlogene) [inline x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: it is default behaviour, you do not need to declare `src_install()` at all.

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild (hunk 1 above)
Comment (lucianposton) [reply x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: removed in current version. it was from -r1 for epatch

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild (hunk 2 above)
Comment (lucianposton) [reply x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: fixed

File: x11-misc/gmrun/gmrun-0.9.2-r2.ebuild (hunk 3 above)
Comment (lucianposton) [reply x11-misc/gmrun/gmrun-0.9.2-r2.ebuild]: removed src_install

**Comment by flappyports** [comment]
Please fix all trailing whitespace and the extra line at the EOF.

**Comment by lucianposton** [comment]
@flappyports Can you provide inline comments so I know which lines you're referring to? The only trailing whitespace or extra line before EOF I see is part of the git generated patch file, not ebuild.

**Pull request closed without merging**